core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -147,7 +147,7 @@ public static void setupContent() throws FileNotFoundException {
 
     @Test
     public void testIsRunning() {
-        try (GenericContainer container = new GenericContainer()) {
+        try (GenericContainer container = new GenericContainer().withCommand("sleep 60")) {
```

Comment:
I would use "top" instead of "sleep 60" because it's continuous (will never exit)

Reply:
Good suggestion - done